typescript/cli/src/commands/options.ts

```diff
@@ -123,7 +123,7 @@ export const dryRunOption: Options = {
   type: 'string',
   description:
     'Chain name to fork and simulate deployment. Please ensure an anvil node instance is running during execution via `anvil`.',
-  alias: ['d'],
+  alias: 'd',
```

Comment:
I believe this had brackets because @yorhodes and I decided to have both `-d` and `-dr` for the `--dry-run` shorthand. Can we add that back in?

Reply:
Yeah I removed the -dr alias because I've seen weird quirks in yargs for aliases longer than one character. Are you okay with just having the one alias?